client/my-sites/themes/theme-options.js

```diff
@@ -59,7 +59,7 @@ const activate = {
 const activateOnJetpack = {
 	label: i18n.translate( 'Activate' ),
 	header: i18n.translate( 'Activate on:', { comment: 'label for selecting a site on which to activate a theme' } ),
-	action: activateWpcomThemeOnJetpack,
+	action: ( themeId, siteId, ...args ) => installAndActivate( themeId + '-wpcom', siteId, ...args ),
```

Comment:
Line of explanation in this place about `-wpcom` would be nice - I know that this is provided in action but would help read code faster. You could point to the action jdoc for clarification.